internal/workload/wrapper.go

```diff
@@ -92,6 +99,10 @@ func (ww Workload) Remove(workloadName string) error {
 	if err := ww.workloads.Remove(id); err != nil {
 		return err
 	}
+	// Remove the service configuration from the system:
+	if err := ww.removeService(workloadName); err != nil {
```

Comment:
That's a good question. You are right, that much safer would be to remove the service first, because it calls stop before and remove the system configuration later and so that correct way of doing things. Thanks for catching this!